package/evtest/package

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(evtest)
+pkgdesc="Kernel evdev device information and monitor"
+url=https://gitlab.freedesktop.org/libevdev/evtest
+pkgver=1.34-1
+timestamp=2020-12-30T02:52Z
+section=utils
+maintainer="Linus K. <linus@cosmos-ink.net>"
+license=GPL-2.0-only
+
+image=base:v1.2.1
+source=(https://gitlab.freedesktop.org/libevdev/evtest/-/archive/evtest-1.34/evtest-evtest-1.34.zip)
+sha256sums=(62f7e34c5bab91b5015de5b056d79051c677c5bd5702facb2885f8e4ba0df84c)
+
+build() {
+    apt-get update
+    apt-get install -y autoconf
```

Comment:
In the absence of `builddepends` (#85) this should be in `prepare`
```suggestion
prepare() {
    apt-get update
    apt-get install -y autoconf
}
build() {
```

Reply:
Oh right, `prepare` doesn't run in the docker container, and `build` does. So it has to be in build. Ignore this.